openc3/lib/openc3/accessors/json_accessor.rb

```diff
@@ -43,7 +51,7 @@ def self.write_item(item, value, buffer)
 
       # Update buffer
       if String === buffer
-        buffer.replace(JSON.generate(decoded, :allow_nan => true))
+        buffer.replace(JSON.generate(decoded.as_json, :allow_nan => true))
```

Comment:
Create additions?

Reply:
Not necessary on generate, only on parse